packages/nextjs/app/builders/0x7f6Ba565e35BB23C307992feBD841D5b2d8c8011/page.tsx

```diff
@@ -0,0 +1,117 @@
+            alt="Avatar"
+            className="rounded-full w-32 h-32 mb-4"
+          />
+          <span className="text-2xl font-bold mb-2">My name is Kong</span>
+          {/* 添加欢迎语和地址 */}
+          <span className="text-2xl font-bold mb-2">Welcome to my page!</span>
+          <span className="text-lg mb-4">This is my personal page on Batch #18.</span>
+          {/* 使用 Address 组件显示地址 */}
+          <span className="text-lg font-bold">My Ethereum Address:</span>
+          <Address address="0x7f6Ba565e35BB23C307992feBD841D5b2d8c8011" onlyEnsOrAddress />
+          {/* 添加关于我的信息和联系方式 */}
+          <div className="text-2xl font-bold mt-3 mb-3">About Me</div>
+          <span className="text-xl max-w-fit font-bold mb-4 mt-2 pr-20 pl-20">
+            I am a backend development engineer with 5 years of work experience, specializing in Java and Python for web
+            development.
+          </span>
+          <span className="text-xl max-w-fit font-bold mb-4 mt-2 pr-20 pl-20">
+            My passion lies in delving deep into development technologies, constantly exploring new solutions to enhance
+            system performance and functionality.
+          </span>
+          <div className="text-2xl font-bold mt-3 mb-3">Contact Me</div>
+          <div className="flex">
+            {/* 添加社交媒体链接 */}
+            <span className="text-lg font-bold pl-20 pr-20">
+              <Link href="https://github.com/fjkang" target="_blank">
+                <svg
+                  className="w-10 h-10"
+                  fill="#ffffff"
+                  viewBox="-51.2 -51.2 614.40 614.40"
+                  xmlns="http://www.w3.org/2000/svg"
+                  transform="matrix(1, 0, 0, 1, 0, 0)"
+                  stroke="#ffffff"
+                >
+                  <g id="SVGRepo_bgCarrier" stroke-width="0"></g>
+                  <g id="SVGRepo_tracerCarrier" stroke-linecap="round" stroke-linejoin="round"></g>
+                  <g id="SVGRepo_iconCarrier">
+                    <title>ionicons-v5_logos</title>
+                    <path d="M256,32C132.3,32,32,134.9,32,261.7c0,101.5,64.2,187.5,153.2,217.9a17.56,17.56,0,0,0,3.8.4c8.3,0,11.5-6.1,11.5-11.4,0-5.5-.2-19.9-.3-39.1a102.4,102.4,0,0,1-22.6,2.7c-43.1,0-52.9-33.5-52.9-33.5-10.2-26.5-24.9-33.6-24.9-33.6-19.5-13.7-.1-14.1,1.4-14.1h.1c22.5,2,34.3,23.8,34.3,23.8,11.2,19.6,26.2,25.1,39.6,25.1a63,63,0,0,0,25.6-6c2-14.8,7.8-24.9,14.2-30.7-49.7-5.8-102-25.5-102-113.5,0-25.1,8.7-45.6,23-61.6-2.3-5.8-10-29.2,2.2-60.8a18.64,18.64,0,0,1,5-.5c8.1,0,26.4,3.1,56.6,24.1a208.21,208.21,0,0,1,112.2,0c30.2-21,48.5-24.1,56.6-24.1a18.64,18.64,0,0,1,5,.5c12.2,31.6,4.5,55,2.2,60.8,14.3,16.1,23,36.6,23,61.6,0,88.2-52.4,107.6-102.3,113.3,8,7.1,15.2,21.1,15.2,42.5,0,30.7-.3,55.5-.3,63,0,5.4,3.1,11.5,11.4,11.5a19.35,19.35,0,0,0,4-.4C415.9,449.2,480,363.1,480,261.7,480,134.9,379.7,32,256,32Z"></path>
+                  </g>
+                </svg>
```

Comment:
I think these SVGs could be stored in a separate file, since they take up too much space in the code.